src/Components/Header.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import MicIcon from '@mui/icons-material/Mic';
```

Comment:
- [ ] I installed all the dependencies but still getting error : 
<img width="1414" alt="Screenshot 2022-10-13 at 21 06 57" src="https://user-images.githubusercontent.com/77038610/195672919-5d95b17d-03b9-443a-b401-45be0f2f1782.png">

Reply:
I added the command that solves these problems to the README.